pkgs/top-level/all-packages.nix

```diff
@@ -7425,6 +7425,7 @@ with pkgs;
   ansible_2_2 = callPackage ../tools/admin/ansible/2.2.nix {};
   ansible_2_3 = callPackage ../tools/admin/ansible/2.3.nix {};
   ansible_2_4 = callPackage ../tools/admin/ansible/2.4.nix {};
+  ansible_2_5 = callPackage ../tools/admin/ansible/2.5.nix {};
```

Comment:
Why do we have all of these versions?

/cc @FRidh

Reply:
We should clean up some older versions. I have no idea which ones to drop, that the maintainers should look into.